bitrise.yml

```diff
@@ -1463,7 +1463,103 @@ workflows:
             App|${BITRISE_APP_URL}
             #mobile-testeng|https://mozilla.slack.com/archives/C02KDDS9QM9
             Mana|https://mana.mozilla.org/wiki/display/MTE/Mobile+Test+Engineering
-
+  Firebase_Performance_Test:
+    before_run:
+    - 1_git_clone_and_post_clone
+    - 2_certificate_and_profile
+    - 3_provisioning_and_npm_installation
+    steps:
+    - xcode-archive@4.7.2:
+        inputs:
+        - compile_bitcode: 'no'
+        - upload_bitcode: 'no'
+        - team_id: 9G8J6YA743
+        - export_method: development
+        - distribution_method: development
+        - export_development_team: 9G8J6YA743
+        - output_tool: xcodebuild
+        - scheme: Fennec_Enterprise
+        - configuration: Fennec_Enterprise
+        - project_path: "$BITRISE_PROJECT_PATH"
+    - deploy-to-bitrise-io@2: {}
+    - script@1.1:
+        title: Firebase Build for Testing Physical Devices
+        inputs:
+        - content: |
+            set -euxo pipefail
+            echo "-- build-for-testing --"
+            mkdir DerivedData
+            xcodebuild -resolvePackageDependencies -onlyUsePackageVersionsFromResolvedFile
+            xcodebuild build-for-testing -project /Users/vagrant/git/Client.xcodeproj \
+                -scheme Fennec_Enterprise \
+                -configuration Fennec_Enterprise \
+                -sdk iphoneos \
+                -destination "generic/platform=iOS" \
+                -derivedDataPath "/Users/vagrant/git/DerivedData" \
+                DEVELOPMENT_TEAM="Mozilla Corporation" \
+                CODE_SIGN_STYLE=Manual \
+                CODE_SIGN_IDENTITY="iPhone Distribution: org.mozilla.ios.Fennec" \
+                PROVISIONING_PROFILE_SPECIFIER="org.mozilla.ios.Fennec_Enterprise"| xcpretty | tee xcodebuild_fennec.log
```

Comment:
Since the profiles are uploaded to Bitrise, when building the app for device, in theory these will be taken from there automatically for the scheme.

Reply:
removed the runtime args to have firebase use defaults